.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+### Expected behavior
+
+### Actual behavior
+
+### Steps to reproduce the behavior
+
```

Comment:
Perhaps something like:

> ### Steps to reproduce the behavior
> 
> **Please provide the steps to reproduce the issue in great detail**. The more detailed your report, the more likely our team will be able to reproduce and and resolve the problem.
> 
> 1. Example step one: Update if statement from $fromhost-ip to $hostname
> 1. Example step two: Restart rsyslog
> 1. Example step three: Note how the debug log confirms how the if statement fails to match the remote hostname and as result the message is not written to the 

The above scenario has the "Example step X" prefix just in case a user forgets to trim an example step from their submission. If they forget and the example step retains the prefix, it would hopefully be less confusing to anyone reviewing the report whether the submitter left an example behind or was intentionally reporting that step.

If using placeholder text:

> ### Steps to reproduce the behavior
> 
> **Please provide the steps to reproduce the issue in great detail**. The more detailed your report, the more likely our team will be able to reproduce and and resolve the problem.
> 
> 1. Insert step one here. Please use lots of detail
> 1. Insert step two here, also using great detail
> 1. Insert any additional steps as needed